git-hub

```diff
@@ -423,16 +423,16 @@ class RequestManager:
     # Open an URL in an authenticated manner using the specified HTTP
     # method. It also add other convenience headers, like Content-Type,
     # Accept (both to json) and Content-Length).
-    def auth_urlopen(self, url, method, body):
-        req = urllib2.Request(url, body)
+    def auth_urlopen(self, url, method, data):
+        req = urllib.request.Request(url, _encode(data))
```

Comment:
I think the change related to `urllib` should go in the commit `Remove Python 3-fix that snuck in` as it isn't what the commit `POST data should be bytes` tries to address

Reply:
Yeah sorry, I removed that in https://github.com/sociomantic-tsunami/git-hub/pull/272/commits/0720d88b681cde602b5f4dffa5c399d4ffad9935